docs/user/README.md

```diff
@@ -38,25 +38,29 @@ file a bug in [Bugzilla in Data Platform & Tools :: Glean.js][gleanjs-bugs].
 > if you want to use Glean.js and is missing some key Glean feature.
 ## Sections
 
-### [Using Glean](./user/index.html)
+### [User's Guides](./user/adding-glean-to-your-project.html)
+
+This section of the book contains mostly step-by-step guides and essays detailing how to
+achieve specific tasks with Glean.
+
+It contains guides on the first steps of integrating Glean into your project,
+choosing the right metric type for you, debugging products that use Glean and
+Glean's built-in error reporting mechanism.
 
-In this section we describe how to use Glean in your own libraries and applications.
-It explains the first steps of integrating Glean into your project, choosing the right metric type for you,
-debugging products that use Glean and Glean's built-in error reporting mechanism.
 If you want to start using Glean to report data, this is the section you should read.
 
-### [Metric Types](./user/metrics/index.html)
+### [API Reference](./reference/yaml/index.html)
+
+This section of the book contains reference pages for Glean’s user facing APIs.
+
+If you are looking for information a specific Glean API, this is the section you should check out.
 
-This sections lists all the metric types provided by Glean, with examples on how to define them
-and record data using them. Before diving into Glean's metric types details, don't forget to
-read the [Choosing a metric type](https://mozilla.github.io/glean/book/user/adding-new-metrics.html#choosing-a-metric-type) page.
+### [Implementation Specific Information](./impl/android/index.html)
```

Comment:
What about:

- Language binding-specific information?
- Platform-specific information? (might be a stretch, because this is not really a platform)

Reply:
Hm, in retrospect I prefer Language Binding. For what it is worth, Glean.js is very much a language binding. That is a strech for Firefox Desktop, which should have it's own subsection here, but I think Firefox Desktop will always be stand out of pattern in all of the book, so that should be fine :)